File: leetcode/easy/1374. Generate a String With Characters That Have Odd Counts/solution.cc
#include <string>

using namespace std;

class Solution {
public:
    string generateTheString(int n) {
        string s;
        if (n % 2 == 0) {
            s += 'a';
            n -= 1;
        }
        while (n--) {
            s += 'b';
        }
        return s;
    }
};

int main() {
    return 0;
}

File: leetcode/easy/1374. Generate a String With Characters That Have Odd Counts/solution.js
// https://leetcode.com/problems/generate-a-string-with-characters-that-have-odd-counts/
/**
 * @param {number} n
 * @return {string}
 */
function generateTheString(n) {
    let res = '';
    if (n % 2 === 0) {
        res += 'a';
        n -= 1;
    }
    while (n--) {
        res += 'b';
    }
    return res;
};